templates/terraform/pre_delete/compute_disk_resource_policies_attachment.go.erb

```diff
@@ -1,7 +1,16 @@
 obj = make(map[string]interface{})
 
+zone, err := getZone(d, config)
+if err != nil {
+	return err
+}
+
+// resourceProjects are added by zone, read by region, i.e.
 // projects/{project}/regions/{region}/resourcePolicies/{resourceId}
-region := getRegionFromZone(d.Get("zone").(string))
+region := getRegionFromZone(zone)
+if region == "" {
+	return fmt.Errorf("unable to infer region from empty zone")
```

Comment:
Seems a bit strange to assume that zone was empty if region isn't present. What if zone is just a bad string?

Reply:
will fix the message